lib/IAMClient.js

```diff
@@ -545,8 +545,9 @@ class IAMClient {
                 return callback(null, obj, res.statusCode, res.statusMessage);
             }
             // Load the error from errors(arsenal)
-            return callback(
-                errors[obj.ErrorResponse.Error.Code], null, res.statusCode);
+            const retErr = obj.ErrorResponse
```

Comment:
ok, got it. But I'm interested in the case of having obj equal to ```{}``` which seems to be possible per getObj() code.